storage/aptosdb/src/db/include/aptosdb_writer.rs

```diff
@@ -265,6 +265,12 @@ impl AptosDB {
                     .commit_transactions(chunk.first_version, chunk.transactions, skip_index_and_usage)
                     .unwrap()
             });
+            s.spawn(|_| {
+                self.ledger_db
+                    .transaction_db()
+                    .commit_transaction_summaries(chunk.first_version, chunk.transactions)
+                    .unwrap()
+            });
```

Comment:
it's the same db with transaction (transaction_db), I'm not sure it's worth it to split out -- we need to deal with truncation separately. I'd just put inline under `commit_transactions`. 
 (I don't think commit_transactions is the bottleneck.) @grao1991 wdyt?

Reply:
It was.

Reply:
I think the state kv is the bottleneck. 
But anyways if they are in the same DB so inherently sequential, making them parallel doesn't really help more than the chunking we are already doing.

As I'm looking at compaction / write amplification, I think we either do the same thing, or split the new thing out to a new DB instance.

Reply:
Well, only WAL is sequential, other things are not. e.g. https://github.com/aptos-labs/aptos-core/pull/10681 made things 10% slower.

What's the problem to store them in a new DB instance again?

Reply:
just the effort. I'm saying either we don't bother to do anything fancy or we add a db instance. 
current implementation is separate thread but still in transaction db.

Reply:
What's the conclusion? Should I insert the transaction summary in the same thread as inserting into TransactionByHashSchema, TransactionSchema? Or should I keep it in separate thread like right now?

Reply:
The transaction is not really the slowest but one of the slower ones.

![image](https://github.com/user-attachments/assets/0f6551e2-22f0-4850-b216-a575192ec7b3)

However to make the new index a separate DB we need to write down a reverse index (version -> account) so that we can to the truncation / pruning properly.

I would do the simplest things for now and observe on the performance test and devnet to see if we want to do the more sophisticated solution directly. (Otherwise we deal with it when we do more optimization for the transaction db).